tcms/testplans/static/testplans/js/get.js

```diff
@@ -428,3 +440,17 @@ function getSelectedTestCases() {
     return tcIds;
 
 }
+
+function getConfirmedSelectedTestCases() {
```

Comment:
You can merge this together with getSelectedTestCases and return a complex data structure, see `selectedCheckboxes` in `refactor_testrun_ui` branch. However I'm questioning if this is at all necessary. 

The view creating new test run and its template is designed to warn the users if they've selected unconfirmed cases and there is already visual clues for that anyway. Independent of the fact if we have this function or not if the user tries to create a new TR by selecting unconfirmed cases it won't be possible.